qiskit_finance/circuit/library/payoff_functions/european_call_pricing_objective.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         )
 
         super().__init__(*european_call.qregs, name="ECEV")
-        self._data = european_call.data
+        self.data = european_call.data
```

Comment:
Do you need to make `data` public?

Reply:
`self` in this context is this class which is a QuantumCircuit and the member there being set, i.e data, is in QuantumCircuit. It used to set the private member var of QuantumCircuit, as you can see, which used to work. But there have been changes in 1.0 and that no longer worked - so this now is using the public data setter of QuantumCircuit that it inherits which does some processing as part of the setter. Either way the code before did not determine the visibility as it is a variable of the parent class - the QuantumCircuit. Does that help?

Reply:
Yes, I got it. Thank you.